vchan/libvchan.h

```diff
@@ -47,6 +47,18 @@ libvchan_t *libvchan_server_init(int domain, int port, size_t read_min, size_t w
 
 libvchan_t *libvchan_client_init(int domain, int port);
 
+/* An alternative path for client connection:
+ * 1. Call libvchan_client_init_async first.
+ * 2. Call libvchan_client_init_fd_for_select() and wait for the FD to be readable.
```

Comment:
Should `libvchan_client_init_async()` return the file descriptor via an out parameter?  That saves a call to `libvchan_client_init_fd_for_select()`.

Reply:
Makes sense.